assembl/scripts/ini_files.py

```diff
@@ -198,6 +198,21 @@ def combine_ini(config, overlay, adding=True):
     return config
 
 
+def migrate_beaker_config(random_ini, overlay):
+    """Migrate old-style session... to beaker.session..."""
```

Comment:
Could I have more context about beaker, what it is, how is it different to previous session system and why we change? I'm a bit lost here...